avm/res/network/private-dns-zone/aaaa/version.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'd suggest to remove all version files from the child modules. We're not supporting versioning them at this time

Reply:
Removed :)